ingest/mappings/publication.json

```diff
@@ -167,7 +188,7 @@
         }
       },
       "publicationYear":{
-        "type":"long"
+        "type":"string"
```

Comment:
Why change publication year to string?  Have we confirmed sorting still works as expected?